### PR TITLE
Glyph rendering fixin

### PR DESCRIPTION
**Pull request by codestreamtest**
Glyphs should render with colored icons in the gutter based on state.12

### CODE REVIEW

File: api/sessionEvents.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { Uri } from "vscode";
 import { memoize } from "../system";
 import { CodeStreamSession, Post, SessionSignedOutReason, SessionStatus } from "./session";
 
+// upadating some codz
```
Comment (planteater) [inline api/sessionEvents.ts:13]: Single from CM

**Comment by planteater** [comment]
delete me!

File: agent/agentConnection.ts [hunk 2]
```diff
@@ -443,6 +443,13 @@ export class CodeStreamAgentConnection implements Disposable {
 			});
 		}
 
+		fetchMyBeer(streamId: string, parentPostId: string) {
+			return this._connection.sendRequest(FetchPostRepliesRequestType, {
```
Comment (planteater) [inline agent/agentConnection.ts:447]: patch this! single

Comment (planteater) [reply agent/agentConnection.ts:447]: rep pathc sing

File: agent/agentConnection.ts [hunk 3]
```diff
@@ -443,6 +443,13 @@ export class CodeStreamAgentConnection implements Disposable {
...
+				streamId: streamId,
+				postId: parentPostId
```
Comment (planteater) [inline agent/agentConnection.ts:449]: now rev 3

**Review by planteater — APPROVED** [review]
gitterdone 3

**Review by planteater — COMMENTED** [review]
gitter 4

File: agent/agentConnection.ts [hunk 4]
```diff
@@ -443,6 +443,13 @@ export class CodeStreamAgentConnection implements Disposable {
...
+			});
+		}
```
Comment (planteater) [inline agent/agentConnection.ts:451]: add 4

**Review by planteater — COMMENTED** [review]
sub 4

**Comment by planteater** [comment]
dun!

**Comment by planteater** [comment]
Closing time, again.

File: configuration.ts [hunk 5]
```diff
@@ -14,12 +14,23 @@ import { Config } from "./config";
...
 const emptyConfig: any = new Proxy<any>({} as Config, {
 	get() {
 		return emptyConfig;
 	}
 });
 
+export interFoo ConfigurationWillChangeEvent {
+	change: ConfigurationChangeEvent;
+	transform?(e: ConfigurationChangeEvent): ConfigurationChangeEvent;
+}
```
Comment (planteater) [inline configuration.ts:32]: check it

**Review by planteater — CHANGES_REQUESTED** [review]
nonono

**Review by planteater — COMMENTED** [review]
blah bleh

**Review by planteater — APPROVED** [review]
ok lgtm

**Review by bcanzanella — COMMENTED** [review]
aaa

**Review by bcanzanella — COMMENTED** [review]
aa

**Review by bcanzanella — COMMENTED** [review]
lgtm

**Review by bcanzanella — APPROVED** [review]
lgtm

**Review by planteater — COMMENTED** [review]
go away now?

File: configuration.ts [hunk 6]
```diff
@@ -14,12 +14,23 @@ import { Config } from "./config";
 import { extensionId } from "./constants";
 import { Functions } from "./system";
 
+const emptyBeers: any = new Proxy<any>({} as Config, {
```
Comment (planteater) [inline configuration.ts:17]: Single before review

Comment (planteater) [reply configuration.ts:17]: rep before

File: configuration.ts [hunk 7]
```diff
@@ -14,12 +14,23 @@ import { Config } from "./config";
...
+	get() {
+		return emptyConfig;
```
Comment (planteater) [inline configuration.ts:19]: And now starting da review

**Review by planteater — COMMENTED** [review]
And now submit da review

File: configuration.ts (hunk 6 above)
Comment (planteater) [reply configuration.ts:17]: rep after

File: configuration.ts [hunk 8]
```diff
@@ -14,12 +14,23 @@ import { Config } from "./config";
...
+	}
+});
+
 const emptyConfig: any = new Proxy<any>({} as Config, {
 	get() {
 		return emptyConfig;
 	}
 });
 
+export interFoo ConfigurationWillChangeEvent {
```
Comment (planteater) [inline configuration.ts:29]: singlllllle

**Comment by planteater** [comment]
Comment with a pic!

WIP MR

**Comment by planteater** [comment]
commount

```
			Container.session.onDidChangeSessionStatus(this.onSessionStatusChanged, this)

```
								
[providers/codeActionProvider.ts](https://github.com/planteater/hello-ny/blob/a5eb6a2ca5b27999009f3a30ef63aa705bf63d90/providers/codeActionProvider.ts#L45) (Line 45)

**Comment by planteater** [comment]
gi

File: agent/agentConnection.ts (hunk 2 above)
Comment (planteater) [inline agent/agentConnection.ts:447]: Comment on 446-447 in diff

File: agent/agentConnection.ts [hunk 9]
```diff
@@ -443,6 +443,13 @@ export class CodeStreamAgentConnection implements Disposable {
 			});
 		}
 
+		fetchMyBeer(streamId: string, parentPostId: string) {
```
Comment (planteater) [inline agent/agentConnection.ts:446]: Comment on line 446

**Pull request closed without merging**